src/frontend/screens/Settings/components/CustomWineProton.tsx

```diff
@@ -33,8 +32,8 @@ export default function CustomWineProton() {
         properties: ['openFile'],
         title: t('box.customWine', 'Select the Wine or Proton Binary')
       })
-      .then(({ path }: Path) => {
-        if (!customWinePaths.includes(path)) {
+      .then((path) => {
+        if (path && !customWinePaths.includes(path)) {
```

Comment:
do we want this `path &&` here? if this is true then the `path ? [...customWinePaths, path] : customWinePaths` at line 38 will always use the true condition

maybe it should be something like:

```ts
if (!path) {
  setCustomWinePaths(customWinePaths)
} else if (!customWinePaths.includes(path)) {
  setCustomWinePaths([...customWinePaths, path])
}
```

or something like that

Reply:
Well, `setCustomWinePaths(customWinePaths)` is essentially a no-op (not sure if that would trigger a re-render as well), so I'm just going to remove that 2nd ternary operator there & always use `[...customWinePaths, path]`